ingestion_tools/scripts/data_validation/tests/annotation/test_segmentationmask_annotation.py

```diff
@@ -0,0 +1,168 @@
+from typing import Dict
+
+import numpy as np
+import pytest
+from mrcfile.mrcinterpreter import MrcInterpreter
+from tests.helper_mrc import HelperTestMRCHeader
+
+ZATTRS_AXIS_ORDER = ["z", "y", "x"]
+
+
+# By setting this scope to session, scope="session" fixtures will be reinitialized for each run + voxel_spacing combination
+@pytest.mark.annotation
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestSegmentationMask(HelperTestMRCHeader):
+
```

Comment:
in the fixture below (set_class_variables), we don't actually use the fixture, but instead it gets called to set some class variables of HelperTestMRCHeader that gets used by itself